services_app/src/main/java/org/opendatakit/services/preferences/fragments/SettingsMenuFragment.java

```diff
@@ -61,7 +61,7 @@ public boolean onPreferenceClick(Preference preference) {
   public void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
 
-    preferenceViewModel = ViewModelProviders
+    preferenceViewModel = new ViewModelProvider
```

Comment:
Not sure how this works since the new doesn't have ".of" please test and compile your code before submitting